### PR TITLE
BETA: Register duyeric.is-a.dev

### PR DESCRIPTION
**Pull request by duyeric**
Added `duyeric.is-a.dev` using the site.

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please use a valid record.

File: domains/duyeric.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+        "owner": {
+           "username": "duyeric",
+           "email": "duyericmmo@gmail.com"
+        },
+        "record": {
+            "CNAME": "www"
```
Comment (WilliamDavidHarrison) [inline domains/duyeric.json:7]: Please use a valid record.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/duyeric.json (hunk 1 above)
Comment (duyeric) [reply domains/duyeric.json:7]: "A": 172.245.139.131

**Comment by duyeric** [comment]
"A": 172.245.139.131
Thanks

Vào CN, 23 thg 7, 2023 vào lúc 13:17 William Harrison <
***@***.***> đã viết:

**Review by wdhdev — APPROVED** [review]

**Pull request merged**